### PR TITLE
RELATED: RAIL-272 kpi with numberjs

### PR DESCRIPTION
**Pull request by zbycz**
+ https://github.com/gooddata/gdc-numberjs/pull/45

### CODE REVIEW

File: src/components/simple/Kpi.tsx [hunk 1]
```diff
@@ -81,8 +81,10 @@ export class Kpi extends React.Component<IKpiProps, null> {
         );
     }
 
-    private getFormattedResult(result: string): string {
-        return numeral(result).format(this.props.format);
+    private getFormattedResult(result: string) {
+        const formattedNumber = numberFormat(parseFloat(result), this.props.format);
+        const { label, color } = colors2Object(formattedNumber);
+        return color ? <span style={{ color }}>{label}</span> : label;
```
Comment (zbycz) [inline src/components/simple/Kpi.tsx]: Maybe should i merge the <span>s together?

**Comment by zbycz** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/192/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-el7/824/ : SUCCESS in 1m 54s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/139/ : SUCCESS in 2m 26s (non-voting)

**Comment by zbycz** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/193/ : SUCCESS in 21s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-el7/825/ : FAILURE in 12m 35s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/140/ : ABORTED in 36m 13s (non-voting)

**Comment by zbycz** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/195/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-el7/827/ : SUCCESS in 1m 59s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/142/ : SUCCESS in 6m 43s (non-voting)

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-merge-compliance-check/85/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-el7/828/ : SUCCESS in 1m 01s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/143/ : FAILURE in 4m 09s (non-voting)

**Pull request merged**